src/parser/parsers/heroes.py

```diff
@@ -227,7 +236,9 @@ def _get_dps_stats(self, weapon_stats):
             'ReloadDelay': weapon_stats['ReloadDelay'],
             'ReloadTime': weapon_stats['ReloadTime'],
             'ClipSize': weapon_stats['ClipSize'],
-            'RoundsPerSecond': weapon_stats['RoundsPerSecond'],
+            'RoundsPerSecond': weapon_stats['RoundsPerSecondAtMaxSpin']
```

Comment:
This has skewed the dps way higher than it actually is, as it implies mcginnis fires at max spin all the time. Instead the spin stats should be separate to this and just considered in the DPS calculation

Reply:
I just saw this in the description 

> Only "RoundsPerSecondAtMaxSpin" is used in dps calculation because the DPS calc assumes already being at max spin.

Which DPS calc assumes it, the one in game?

Reply:
I'm not sure which one it displays in game, it might be dynamic to how revved up you are. 

I've intentionally set it up to use fully revved up RPS because its up to 2.5x RPS and after only 2 seconds of firing, making fully-revved up dps much more accurate as a reference to her average dps than non-revved up. 

> Instead the spin stats should be separate to this and just considered in the DPS calculation
The spin stats already only factor into DPS. If users want to know her un-revved dps, they can quickly determine this as well.